### PR TITLE
Fetch Transaction

### PR DESCRIPTION
**Pull request by youthtrouble**
i added FetchTransaction using Transaction ID, please check and test, I also updated the README.md and main.go to fit.

### CODE REVIEW

File: payment.go [hunk 1]
```diff
@@ -36,6 +36,16 @@ type chargeCard struct {
 	Reference string
 }
 
+type fetchTransaction struct {
+	Id		  int64
```
Comment (ichtrojan) [inline payment.go]: Not formatted

File: payment.go [hunk 2]
```diff
@@ -200,3 +210,61 @@ func (config paystackConfig) ChargeCard(authorization string, email string, amou
...
+
+		_ = response.JSON(&data)
+
+
+		return fetchTransaction{
+			Id:                  data.Data.Id,
+			Status:              data.Data.Status,
+			Amount:              data.Data.Amount,
+			Reference:           data.Data.Reference,
+			Message:   			 data.Data.GatewayResponse,
```
Comment (ichtrojan) [inline payment.go]: Not formatted

File: payment.go [hunk 3]
```diff
@@ -200,3 +210,61 @@ func (config paystackConfig) ChargeCard(authorization string, email string, amou
 
 	return chargeCard{}, errors.New(data.Message)
 }
+
+
+func (config paystackConfig) FetchTransaction(id int64) (fetchTransaction, error) {
```
Comment (ichtrojan) [inline payment.go]: make `id`explicit, change to `transactionId`

File: payment.go [hunk 4]
```diff
@@ -36,6 +36,16 @@ type chargeCard struct {
 	Reference string
 }
 
+type fetchTransaction struct {
```
Comment (ichtrojan) [inline payment.go]: rename to `transaction`

File: README.md [hunk 5]
```diff
@@ -77,6 +77,22 @@ func main () {
     fmt.Printf("%+v\n", chargeCard)
 }
 ```
+* Fetch Transaction
+
+```go
+...
+
+func main () {
+    chargeCard, err := paystack.FetchTransaction(292584114)
```
Comment (ichtrojan) [inline README.md]: change `chargeCard` to `fetchTransaction`

File: payment.go [hunk 6]
```diff
@@ -200,3 +210,61 @@ func (config paystackConfig) ChargeCard(authorization string, email string, amou
...
+			Amount:              data.Data.Amount,
+			Reference:           data.Data.Reference,
+			Message:   			 data.Data.GatewayResponse,
+			Bank:                data.Data.Authorization.Bank,
+			CardType:            data.Data.Authorization.Brand,
+		}, nil
+	}
+
+	return fetchTransaction{}, errors.New(data.Message)
+
```
Comment (ichtrojan) [inline payment.go]: remove unnecessary whitespace

File: README.md [hunk 7]
```diff
@@ -87,6 +103,8 @@ comming soon....
 
 ## Contribution
```
Comment (ichtrojan) [inline README.md]: Rename to `Contributors`

**Comment by ichtrojan** [comment]
I left some comments on the changes made, also, can you amend your commit messages and make them explicit. This will help other contributors understand what you did.

**Comment by youthtrouble** [comment]
Hello,
I've made the changes requested, though I did not fully understand the I formatted comments, please if you could explain better. I might have still done it wrongly now. Please check and give feedback. Thank you.

**Comment by ichtrojan** [comment]
I meant, the formatting of the line in question doesn't follow Go's guidelines.

**Comment by ichtrojan** [comment]
Will fix on my end, looks good to go.
